src/main/java/healthy/lifestyle/backend/users/dto/SignupRequestDto.java

```diff
@@ -35,14 +36,18 @@ public class SignupRequestDto {
     @FullnameValidation
     private String fullName;
 
+    @NotNull private Long countryId;
+
     public SignupRequestDto() {}
 
-    public SignupRequestDto(String username, String email, String password, String confirmPassword, String fullName) {
+    public SignupRequestDto(
```

Comment:
Please refactor this class by adding Lombok, like in other classes. It will save time when we want to add a new field.